### PR TITLE
Enable the install of git 2.15 on all platforms

### PR DESCRIPTION
**Pull request by jdekonin**
Signed-off-by: Joe deKoning <joe_dekoning@ca.ibm.com>

### CODE REVIEW

File: ansible/playbooks/rhel.yml [hunk 1]
```diff
@@ -170,6 +171,8 @@
     #######################
     # Additional Packages #
     #######################
+    #       ccache        #
```
Comment (bblondin) [inline ansible/playbooks/rhel.yml]: ??? bad copy paste?

Comment (jdekonin) [reply ansible/playbooks/rhel.yml]: I can add the extra line of hashes if you want.  I just added this as a section header.

File: ansible/playbooks/rhel.yml [hunk 2]
```diff
@@ -198,55 +201,75 @@
...
         dest: /home/{{ Jenkins_Username }}/git-2.15.0.tar.xz
         mode: 0440
       when:
-        - ansible_distribution_major_version == "7"
-        - ansible_architecture == "ppc64le"
+        - git_version.stdout | version_compare('2.0', '<=', strict=True)
       tags: git-src
 
+    - stat:
+        path=/home/{{ Jenkins_Username }}/git-2.15.0.tar.xz
```
Comment (bblondin) [inline ansible/playbooks/rhel.yml]: Why are we using home and not temp? /tmp

File: ansible/playbooks/rhel.yml [hunk 3]
```diff
@@ -198,55 +201,75 @@
...
         copy: False
       when:
-        - ansible_distribution_major_version == "7"
-        - ansible_architecture == "ppc64le"
+        - git_download.stat.exists == True 
       tags: git-src
 
     - name: Compile and install git 2.15
-      shell: cd /home/{{ Jenkins_Username }}/git-2.15.0 && make prefix=/usr all && make prefix=/usr install
+      shell: cd /home/{{ Jenkins_Username }}/git-2.15.0 && ./configure --prefix=/usr --without-tcltk && make && make install
```
Comment (bblondin) [inline ansible/playbooks/rhel.yml]: Should be compiling in parallel
cd /home/{{ Jenkins_Username }}/git-2.15.0 && ./configure --prefix=/usr --without-tcltk && ``` make -j {{ ansible_processor_vcpus }} ``` && make install

**Comment by jdekonin** [comment]
Requested changes made.

File: ansible/playbooks/rhel.yml [hunk 4]
```diff
@@ -198,55 +201,75 @@
         - ansible_architecture == "S390"
       tags: ccache
 
+    #########
+    #  GIT  #
+    #########
+    - name: Check git version installed
+      shell: git --version | sed -e 's/git version //g' | cut -c 1-3
```
[the remark this answers is not in the document]
Comment (jdekonin) [reply ansible/playbooks/rhel.yml]: Git is a package added in "Install Build tools", but I added the change.

**Pull request merged**